### DISCUSSION TITLE
Uncaught (in promise) TypeError: Alpine.navigate is not a function

### DISCUSSION THREAD

**Discussion by devhoussam1998** [livewire/livewire#5901] [1. Help]
<img width="1157" alt="Screenshot 2023-07-23 at 15 54 07" src="https://github.com/livewire/livewire/assets/121255432/8d341198-fea5-4f21-b1b4-612bb520094c">

**Comment by joshhanley** [comment]
@devhoussam1998 thanks for reporting! Can you share a screenshot of the stack trace and also what you did to trigger the error? Thanks!

**Reply by Delta-40** [reply]
Edit:
Just figured out that it was actually because of the because "laravel-frontend-presets/tall" that imports Alpinejs with the vite setup. 
When I removed the
```javascript
import Alpine from 'alpinejs'
window.Alpine = Alpine
Alpine.start()
``` 
from resources/js/app.js, it started working straight away again. 

@devhoussam1998 Try to remove the js code mentioned, and also npm remove alpinejs 👍🏻
